extension/src/XliffFunctions.ts

```diff
@@ -299,68 +295,62 @@ export function refreshSelectedXlfFileFromGXlf(
           }
           refreshResult.numberOfUpdatedNotes++;
         }
-        if (langTransUnit.sourceIsEmpty() && langTransUnit.targetIsEmpty()) {
+        if (
+          langTransUnit.sourceIsEmpty() &&
+          langTransUnit.targetIsEmpty() &&
+          lfSettings.preferLockedTranslations
+        ) {
           langTransUnit.insertCustomNote(
             CustomNoteType.refreshXlfHint,
             RefreshXlfHint.emptySource
           );
-          refreshResult.numberOfReviewsAdded++;
+          langTransUnit.target.translationToken = TranslationToken.review;
```

Comment:
Shouldn't we keep to increase the counter, now that we adds this review token?
```suggestion
          langTransUnit.target.translationToken = TranslationToken.review;
          refreshResult.numberOfReviewsAdded++;
```

Reply:
Hmm yeah. It feels like I might have tricked my self here